docs/index.md

```diff
@@ -50,3 +71,28 @@ TorchServe is a performant, flexible and easy to use tool for serving PyTorch ea
 * [mlflow-torchserve](https://github.com/mlflow/mlflow-torchserve) - Deploy mlflow pipeline models into TorchServe
 * [Kubeflow pipelines](https://github.com/kubeflow/pipelines/tree/master/samples/contrib/pytorch-samples) - Kubeflow pipelines and Google Vertex AI Managed pipelines
 * [NVIDIA MPS](mps.md) - Use NVIDIA MPS to optimize multi-worker deployment on a single GPU
+
+## 📰 News
+* [Torchserve Performance Tuning, Animated Drawings Case-Study](https://pytorch.org/blog/torchserve-performance-tuning/)
+* [Walmart Search: Serving Models at a Scale on TorchServe](https://medium.com/walmartglobaltech/search-model-serving-using-pytorch-and-torchserve-6caf9d1c5f4d)
+* [🎥 Scaling inference on CPU with TorchServe](https://www.youtube.com/watch?v=066_Jd6cwZg)
+* [🎥 TorchServe C++ backend](https://www.youtube.com/watch?v=OSmGGDpaesc)
+* [Grokking Intel CPU PyTorch performance from first principles: a TorchServe case study](https://pytorch.org/tutorials/intermediate/torchserve_with_ipex.html)
```

Comment:
Could we please also add link to Part 2? https://pytorch.org/tutorials/intermediate/torchserve_with_ipex_2.html?highlight=grokking%20pytorch%20intel%20cpu%20performance%20from%20first%20principles%20part

Reply:
Done. Also added in README